src/project-css/database/migrations/2016_11_10_213755_create_collections_table.php

```diff
@@ -15,7 +15,7 @@ public function up()
     {
         Schema::create('collections', function (Blueprint $table) {
             $table->increments('id');
-            $table->string('clctnName')->unique();
+            $table->string('clctnName', 191)->unique();
```

Comment:
According to a guide I found we needed to set the string length and not use the default. I was getting a error when I changed the character encoding type until I set it to 191. The default length wasn't enough to handle the extra bytes.